src/_main_/utils/base_model.py

```diff
@@ -39,3 +39,22 @@ def to_json(self, full=False, tiny_info=False):
 
     class Meta:
         abstract = True
+
+
+class CustomMeta(models.base.ModelBase):
```

Comment:
why are you declaring the class?

Simply declare the inner class in all the models.

Reply:
Do you mean declaring a CustomMeta class in the models ?